bin/juttle-engine

```diff
@@ -7,7 +7,7 @@ var engine = require('..');
 
 var defaults = {
     'port': 8080,
-    'root': '/',
+    'root': process.cwd(),
```

Comment:
Should be `__dirname` so its not dependent on where `juttle-engine` is run from.